.github/workflows/ci.yml

```diff
@@ -25,22 +49,37 @@ jobs:
           ssh-key: ${{ secrets.ACTIONS_DEPLOY_KEY }}
           submodules: recursive
 
+      - uses: actions/download-artifact@v3
+        with:
+          path: ${{ github.workspace }}/artifacts
+
+      - name: Prepare Arduino Core
+        run: |
+          ls -R ${{ github.workspace }}/artifacts
```

Comment:
Probably don't need to leave the `ls` calls in this job for the final submit (leftover debug code?)